compiler/src/dmd/e2ir.d

```diff
@@ -7028,12 +7028,22 @@ elem *callCAssert(ref IRState irs, const ref Loc loc, Expression exp, Expression
             auto eassert = el_var(getRtlsym(RTLSYM.C__ASSERT_FAIL));
             ea = el_bin(OPcall, TYvoid, eassert, el_params(elmsg, efilename, eline, efunc, null));
         }
+        version (CRuntime_Glibc)
+        {
+            // __assert_fail(exp, file, line, func);
+            elem* efunc = getFuncName();
+            auto eassert = el_var(getRtlsym(RTLSYM.C__ASSERT_FAIL));
+            ea = el_bin(OPcall, TYvoid, eassert, el_params(efunc, eline, efilename, elmsg, null));
```

Comment:
So are params passed left-to-right or right-to-left?

This invocation differs from musl, so both can't be right.